learning-outcomes.md

```diff
@@ -0,0 +1,50 @@
+# Week 4 Learning Outcomes
+
+## Node
+
+### Servers
+
+- [ ] **We can create a web server that responds to http requests**
+- [ ] We can conditionally set ports for our server based on the runtime environment
+- [ ] We can read secrets from a .env file
+- [ ] We can prevent sensitive data from being pushed to GitHub
+
+### GET requests
+
+- [ ] We can route requests to the correct handler function(s)
+- [ ] We can serve different types of files to the client
+- [ ] We can read information sent in a URL query string
+
+### POST requests
+
+- [ ] We can handle data received from a POST request
+- [ ] We can ‘chunk’ data sent from the client side on the server side
+- [ ] We can give descriptive names to HTML form input fields so that it is easy to access data in the POST request object
```

Comment:
```suggestion
- [ ] We can give descriptive names to HTML form input fields so that it is easy to access data in the request
```
Forms make `GET` requests by default (see [my new workshop](https://github.com/oliverjam/learn-html-forms)) so this should probably be a more general LO